Project.toml

```
@@ -1,6 +1,6 @@
 name = "Compat"
 uuid = "34da2185-b29b-5c13-b0c7-acf172513d20"
-version = "3.6.0"
+version = "3.7.0"
```

```suggestion
version = "3.8.0"
```
... as the current version has become 3.7.0 in the meantime.